mantidproject/mantid
# Beta fix: Fit to raw data in Muon Analysis

Fix the "fit to raw data" option in the Muon Analysis interface,

**To test:**
_Data set: `MUSR00022725.nxs` in unit test data_
- Set facility to ISIS, open _Interfaces/Muon/Muon Analysis_ and load data set
- Go to "Settings" tab, in "Data Binning" select fixed rebinning with steps of 5 (so the difference between rebinned and raw data is clear)
- Go to "Data Analysis" tab:
  - Function (top part of page): set up `GausOsc`, Frequency 1.3, tick "Global" box for A, Sigma, Frequency
  - Data (centre of page): tick checkboxes for groups `fwd` and `bkwd`, set label for fit to "Rebinned"
  - Options (bottom of page): ensure "Fit to Raw Data" is off
- Fit the data
- Now turn on "Fit to Raw Data" option. Set the fit label to "Raw" and fit again.
- Compare the workspace groups `MuonSimulFit_Rebinned` and `MuonSimulFit_Raw`. They should be different:
  ![results_raw_rebinned](https://cloud.githubusercontent.com/assets/15363125/18843451/f970c620-8410-11e6-8e69-edf64e5a90eb.PNG)

Re #17602 (does not close that issue as it contains other points too)

No release notes. The new fitting functionality is already contained in the 3.8 release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
